test/spec/ssg.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+# example of a frontMatter file for a static site generator
+
+title: '{{title}} version {{version}}'
```

Comment:
maybe this one without tags, but only if you follow my advice in the readme, for the docs. If you have a better idea of how to document them in details, then feel free to leave it as it is